primitives/src/lib.rs

```diff
@@ -385,6 +386,9 @@ macro_rules! create_currency_id {
 					$(CurrencyId::$symbol => $deci,)*
 				}
 			}
+            fn one(&self) -> Balance {
```

Comment:
Minor nit, can you put this under the `impl`, doesn't really fit in `CurrencyInfo` IMO.

Reply:
Also the formatting is slightly off, `cargo fmt` doesn't really fix inside macro definitions.